.changeset/twenty-terms-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Updated backend to write stack trace when the backend fails to start up.
```

Comment:
Could you update this to include instructions for existing apps as well? Similar to other entries in https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md

Reply:
I can do that, although this is somewhat duplicating the backstage upgrade helper.  (e.g. that previosu change will show up in `app-config.yaml` in the upgrade helper https://backstage.github.io/upgrade-helper/?from=1.5.0-next.1&to=1.5.0-next.2

Reply:
Yep idd, we're keeping this duplication for now at least though. Since changelog can provide a lot more context it's a nice place to consume all the changes. Will likely aim to introduce annotations to the upgrade helper so that we can skip changelog entries eventually though